qemu/tests/cfg/nic_hotplug.cfg

```diff
@@ -54,3 +54,7 @@
             type = nic_hotplug
             nics = ""
             extra_params += "-net none"
+        - used_netdev:
```

Comment:
no need. since it's already test with vhost=on according to the log.

-netdev tap,id=idPHRGwY,vhost=on,vhostfd=20,fd=19